src/Discord.Net.Rest/Entities/Channels/RestGuildChannel.cs

```diff
@@ -16,6 +16,8 @@ public class RestGuildChannel : RestChannel, IGuildChannel, IUpdateable
         internal IGuild Guild { get; }
         public string Name { get; private set; }
         public int Position { get; private set; }
+        public ulong? CategoryId { get; private set; }
+        public async Task<ICategoryChannel> GetCategoryAsync() => CategoryId == null ? null : await Guild.GetChannelAsync(CategoryId.Value).ConfigureAwait(false) as ICategoryChannel;
```

Comment:
Probably best to put the body on a newline, to be consistent with the rest of the lib:
```cs
int MyMethod()
    => 5;
```